File: Said_Coley_Romero/Programas/Dart/Trabajo_27_de_Mayo.dart
import 'dart:io';

void main() {
  double rentaAnual = 1500;
  print("¿Cuál es tu renta anual? $rentaAnual");

if (rentaAnual <= 10) {
  print("Tu descuento es de 20%");
} else if (rentaAnual <= 100) {
  print("Tu descuento es de 10%");
} else if (rentaAnual < 1000) {
  print("Tu descuento es de 5%");
} else if (rentaAnual >= 1000 && rentaAnual <= 2000) {
  print("Tu descuento es de 20%");
} else {
print("Tu descuento es de 1%");
 }
}

File: Said_Coley_Romero/Programas/Ruby/Trabajo_27_de_Mayo.rb
renta_anual = 1500

puts "¿Cuál es tu renta anual? #{renta_anual}"

if renta_anual <= 10
    puts "Tu descuento es de 20%"
elsif renta_anual <= 100
    puts "Tu descuento es de 10%"
elsif renta_anual < 1000
    puts "Tu descuento es de 5%"
elsif renta_anual >= 1000 && renta_anual <= 2000
    puts "Tu descuento es de 20%"
else
    puts "Tu descuento es de 1%"
end